ext/node/ops/sqlite/statement.rs

```diff
@@ -537,10 +581,13 @@ impl StatementSync {
     // Reset to return correct change metadata.
     drop(reset);
 
-    Ok(RunStatementResult {
-      last_insert_rowid: db.last_insert_rowid(),
-      changes: db.changes(),
-    })
+    Ok(
+      RunStatementResult {
+        last_insert_rowid: db.last_insert_rowid(),
+        changes: db.changes(),
+      }
+      .serialize(scope, self.use_big_ints.get()),
```

Comment:
Huh - seems these methods are not magical at all. And now I see there's `deno_core::convert::ToV8` that we could use instead. Thoughts?

Reply:
Oh I completely forgot ToV8 and FromV8 existed. I'll update these to use them